webscraper.py

```diff
@@ -0,0 +1,27 @@
+from selenium import webdriver
```

Comment:
Is this ment to be in here? Don't see any commits that mention it, seems kinda out of place for this PR